_pages/fsa/dynamicforms/json-structure.md

```diff
@@ -0,0 +1,472 @@
+  controlName?: string;
+  conditions?: ValidatorFunction[];
+}
+
+export interface LocalizedString {
+  default?: string;
+  [lang: string]: string;
+}
+
+export interface ValidatorFunction {
+  name: string;
+  arguments?: ValidatorArgument[];
+}
+
+export interface ValidatorArgument {
+  value: string;
+}
+
+export interface FieldOption {
+  name: string;
+  label: LocalizedString;
+  selected?: boolean;
+  icon?: string;
+}
+
+```
+Now we will try to explain interface structure in more details:
+
+## FormDefinition 
+Represents top level entity which hold group of form field
+- [formGroups](#dynamicformgroup) - Defines array of form groups
+- formId - Attaches ID to the form
+- cssClass - Enables adding unique CSS class to the form wrapper. More info can be found [Here]({{ site.baseurl }}{% link _pages/fsa/dynamicforms/styling.md %})
+
+## DynamicFormGroup
+Represents group of form controls
+- groupCode - Adds name to the form group which can be found in payload on form submit. In this example of submitted form data object, we can see that form controls are wrapped in two form groups:
+```typescript
+groupOne: 
+    control1: "0"
+groupTwo: 
+    control2: "2020-12-31"
+    control3: "YEARLY"
+```
+- fieldConfigs - Array of controls in this form group
+- cssClass - Enables adding unique CSS class to the form group wrapper. More info can be found [Here]({{ site.baseurl }}{% link _pages/fsa/dynamicforms/styling.md %})
+- dependsOn - With this property, we can define whole group`s visibility by defining control name on which group depends on.
+
+## FieldConfig
+Represents individual control
+- name - Sets control name
+- required - Defines if control is required
+- disabled - Defines if control is disabled ( Disabled fields are not picked up on submit )
+- [label](#localizedstring) - Defines control label ( supports localization)
+- options - Defines select`s, radio buttons etc.. options
+- cssClass - Enables adding unique CSS class to the form control. More info can be found [Here]({{ site.baseurl }}{% link _pages/fsa/dynamicforms/styling.md %})
+- gridClass - Enables adding Bootstrap class to the form control to create a grid behaviour in the form. More info can be found [Here]({{ site.baseurl }}{% link _pages/fsa/dynamicforms/styling.md %})
+- apiValue - Used with 'dynamicSelect' controll type, to fetch options dynamically from defined URL
+- placeholder - Sets control placeholder text
+- fieldType - Defines type of the field
+- value - Sets default value to control
+- hidden - If set to true, control will be rendered but not visible on the page 
+- [error](#localizedstring) - Defines localized error message fot this control
+- validations - Defines validations 
+- [dependsOn](#dependson) - Here we can define if our control should be visible depending on other controls value
+- [prefillValue](#prefillvalue) - Defines if field should be pre-filled with some data ( user, cart data from SPA store )
+- readonly - Sets readonly property 
```

Comment:
Add here details regarding sending this field on submit, something like: "Field is disabled but it will be sent to the backend on submit" - like you explained for disabled property